docs/search/patterns.md

```diff
@@ -0,0 +1,581 @@
+---
+sidebar_position: 40
+title: Search Patterns
+---
+
+In today's world, a fundamental requirement for running a successful business is to be able to quickly and accurately locate certain information from large data repositories. This is called _indexing_ and _searching_.
+
+GDN brings two main advantages over other data stores:
+
+- GDN is equipped with a state-of-the-art indexing and search facility that enables users to perform sophisticated search operations on [multi-model](https://www.macrometa.com/topics/multi-model-database) data storage including key-value pairs, documents, and graphs.
+  - You do not need to reformat data to make it compatible with GDN search.
+  - GDN search is capable of complex operations such as faceted search and geospatial serach.
+- Any time we update Macrometa GDN, [search indexes](../search/views.md) are automatically updated globally. 
+
+This page provides detailed examples of possible search query patterns. All examples use the GDN Web console and HTTP REST API.
+
+## Example Dataset
+
+This example uses the London-based hotel reviews [dataset](https://github.com/Macrometacorp/datasets/blob/master/hotel-reviews/hotels.json) obtained from [Kaggle](https://www.kaggle.com/PromptCloudHQ/reviews-of-londonbased-hotels). The refined dataset has 10,000 reviews collected from a travel portal.
+
+![Schema of a hotel review](/img/search/schema.jpg)
+
+You can download the dataset [here](https://raw.githubusercontent.com/Macrometacorp/datasets/master/hotel-reviews/hotels.json). After downloading the JSON file, copy its content between the empty array definition of the data item in the following command. The dataset can be imported to your GDN federation by using the CURL command on a terminal as shown in the following example. 
```

Comment:
Please rewrite the first sentence so that the link text is more descriptive than "here".